### PR TITLE
[HORN-1567] add readme file

### PR DESCRIPTION
**Pull request by LukaszStaszewski-TomTom**

### CODE REVIEW

**Review by ArkadiuszMaciaszczyk-TomTom — APPROVED** [review]

File: bot-double-inner-ring/README.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+* both ways are closed,
+* both ways have the same nodes (direction of way digitalization and starting node can be different),
+* duplicating way should have tags,
+* duplicating way is not member of any relation,
+* inner ring way should have no tags,
+* inner ring way is member of only 1 relation (the one from Osmose violation),
+* optional: relation and duplicating way should have required "source" tag (e.g. "source=CanVec 8.0 - NRCan).
+
+When violation is confirmed by BOT, then data modification is generated. Data modification, is understood as:
+* coping tags from way which does not belong to any relation to the way which is a member of violating relation,
```
Comment (BartoszChrominski-TomTom) [inline bot-double-inner-ring/README.md:36]: copying :)

File: bot-double-inner-ring/README.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+>               --run.parameters.consumer-key=user-consumer-key \
+>               --run.parameters.consumer-secret=user-consumer-secret \
+>               --run.parameters.path-to-violations-file=user-file-with-violations \
+
+About other possible parameters, please read `Input parameters` section.
+
+## Run parameters
+### Required parameters to provide
+Following run parameters are required to provide by the user:
+* 	The `token` identifier. It is generated once the consumer completes the authorization process
```
Comment (KamilLucki-TomTom) [inline bot-double-inner-ring/README.md:54]: I think we should specify where this token can be obtained, please add a link

File: bot-double-inner-ring/README.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+* The `path-to-violations-file` is a path to a JSON formatted file, with violations downloaded from [OSMOSE](http://osmose.openstreetmap.fr/en/issues/open?
+  country=canada_alberta&item=1170&source=&class=1&username=&bbox=)
+
+### Optional parameters to provide
+Following run parameters are optional to provide by the user, but it is not recommended changing their default values:
+* The `openstreetmap-api-url` is the URL address of OSM service, which is used to get data for violations validation and also all modifications created by BOT
+  are sent to this URL. Recommended value of that property is: `https://api.openstreetmap.org/api/0.6/`
+>   --run.parameters.openstreetmap-api-url=https://api.openstreetmap.org/api/0.6/
+
+* The `max-violations-per-changeset` is the amount of modifications sent to one changeset to OSM.
```
Comment (KamilLucki-TomTom) [inline bot-double-inner-ring/README.md:75]: I think this reads better
```suggestion
* The `max-violations-per-changeset` is the amount of modifications sent to OSM per changeset.
```

File: bot-double-inner-ring/README.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
+* Introduction
+* Implementation
+* Running the BOT
+* Run parameters
+  * Required parameters to provide
+  * Optional parameters to provide
+  
+
+
```
Comment (KamilLucki-TomTom) [inline bot-double-inner-ring/README.md:9]: I think this new line should be removed

File: bot-double-inner-ring/README.md (hunk 2 above)
Comment (LukaszStaszewski-TomTom) [reply bot-double-inner-ring/README.md:54]: You are right but this I will add in next PR with ExampleClass for authorization

**Review by KamilLucki-TomTom — APPROVED** [review]

**Review by BartoszChrominski-TomTom — APPROVED** [review]

**Review by ArkadiuszMaciaszczyk-TomTom — APPROVED** [review]

**Pull request closed without merging**